cmd/ddev/cmd/pause_test.go

```diff
@@ -51,7 +52,8 @@ func TestCmdPauseContainers(t *testing.T) {
 	// Confirm all sites are stopped.
 	apps = ddevapp.GetActiveProjects()
 	for _, app := range apps {
-		assert.True(app.SiteStatus() == ddevapp.SitePaused, "All sites should be stopped, but %s status: %s", app.GetName(), app.SiteStatus())
+		status, _ := app.SiteStatus()
+		assert.True(status == ddevapp.SitePaused, "All sites should be stopped, but %s status: %s", app.GetName(), status)
```

Comment:
This assertion comment would be better to give the statusDesc, a good place to use it. Maybe this never happens, don't know.

Reply:
See comment below.